auth0_flutter/lib/src/web/auth0_flutter_plugin_real.dart

```diff
@@ -43,15 +43,16 @@ class Auth0FlutterPlugin extends Auth0FlutterWebPlatform {
   Future<void> loginWithRedirect(final LoginOptions? options) {
     final client = _ensureClient();
 
-    final authParams = JsInteropUtils.stripNulls(interop.AuthorizationParams(
-        audience: options?.audience,
-        redirect_uri: options?.redirectUrl,
-        organization: options?.organizationId,
-        invitation: options?.invitationUrl,
-        max_age: options?.idTokenValidationConfig?.maxAge,
-        scope: options?.scopes.isNotEmpty == true
-            ? options?.scopes.join(' ')
-            : null));
+    final authParams = interop.AuthorizationParams(
+            audience: options?.audience,
+            redirect_uri: options?.redirectUrl,
+            organization: options?.organizationId,
+            invitation: options?.invitationUrl,
+            max_age: options?.idTokenValidationConfig?.maxAge,
+            scope: options?.scopes.isNotEmpty == true
+                ? options?.scopes.join(' ')
+                : null)
+        .prepare(options?.parameters);
```

Comment:
Having a helper here for the common usage on `AuthorizationParams` of stripping nulls and then mixing in custom params cleans up the code a lot.